code/modules/vehicles/speedbike.dm

```diff
@@ -24,6 +24,491 @@
 	icon_state = "speedbike_red"
 	overlay_state = "cover_red"
 
+
+// ATMOBILE AND ATMOBILE UNIQUE INTERNAL ITEMS
+
+
+
+/obj/vehicle/space/speedbike/atmos
+	name = "prototype atmos vehicle"
+	desc = "This vehicle possesses unparalleled utility for atmospheric containment and control"
+	icon_state = "atmo_bike"
+	overlay_state = "cover_atmo"
+	var/obj/machinery/portable_atmospherics/scrubber/huge/internal_scubber = null
+	var/obj/item/extinguisher/vehicle/internal_extinguisher = null
+	var/ex_out = FALSE
+	var/obj/machinery/portable_atmospherics/canister/proto/default/oxygen/CAN = null
+	var/loaded = TRUE
+	light_range = 7
+
+/obj/vehicle/space/speedbike/atmos/New()
+	. = ..()
+	internal_scubber = new /obj/machinery/portable_atmospherics/scrubber/huge(src)
+	internal_scubber.on = 1
+	internal_extinguisher = new /obj/item/extinguisher/vehicle(src)
+	internal_extinguisher.vehicle = src
+	CAN = new /obj/machinery/portable_atmospherics/canister/oxygen(src)
+//	CAN = new /obj/machinery/portable_atmospherics/canister/proto/default/oxygen(src)
+
+/obj/item/extinguisher/vehicle
+	name = "extinguisher nozzle"
+	desc = "A heavy duty nozzle attached to a massive reserve tank."
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "atmos_nozzle"
+	item_state = "nozzleatmos"
+	w_class = WEIGHT_CLASS_BULKY
+	safety = FALSE
+	max_water = 1000
+	power = 8
+	cooling_power = 10
+	recoil = 0
+	var/obj/vehicle/space/speedbike/atmos/vehicle = null
+
+/obj/item/extinguisher/vehicle/dropped(mob/user)
+	..()
+	user << "<span class='notice'>The fire hose snaps back into the [src]!</span>"
+	playsound(get_turf(src),'sound/items/change_jaws.ogg', 75, 1)
+	if(vehicle)
+		loc = vehicle
```

Comment:
forceMove

Reply:
I'll try forceMove again but this was tested extensively (300 hours) and I recall forceMove had some critical flaw that allowed the item to either be dropped or get stuck. I had at least one maintainer tell me to use loc here.

Reply:
trust them then